ToolBox/EstimizeDataDownloader/EstimizeDataDownloader.cs

```diff
@@ -159,6 +164,39 @@ protected void SaveContentToFile(string destinationFolder, string ticker, IEnume
             File.WriteAllLines(finalPath, finalLines);
         }
 
+        /// <summary>
+        /// Tries to normalize a potentially defunct ticker into a normal ticker.
+        /// </summary>
+        /// <param name="ticker">Ticker as received from Estimize</param>
+        /// <param name="nonDefunctTicker">Set as the non-defunct ticker</param>
+        /// <returns>true for success, false for failure</returns>
+        public static bool TryNormalizeDefunctTicker(string ticker, out string nonDefunctTicker)
+        {
+            // The "defunct" indicator can be in any capitalization/case
+            if (ticker.IndexOf("defunct", StringComparison.OrdinalIgnoreCase) > 0)
+            {
+                foreach (var delimChar in _defunctDelimiters)
+                {
+                    var length = ticker.IndexOf(delimChar);
+
+                    // Continue until we exhaust all delimiters
+                    if (length == -1)
+                    {
+                        continue;
+                    }
+
+                    nonDefunctTicker = ticker.Substring(0, length).Trim();
```

Comment:
Based on the tests, `Trim` isn't really required

Reply:
I'll update the tests to include a `TICKER -defunct` case. I've seen tickers with a space between the defunct indicator before, hence the `.Trim()`